src/SensioLabs/Connect/Api/Entity/Member.php

```diff
@@ -11,10 +11,13 @@
 
 namespace SensioLabs\Connect\Api\Entity;
 
+@trigger_error(sprintf('The "%s" class is deprecated since Connect 4.3 and will be removed in 5.0.', Member::class), E_USER_DEPRECATED);
```

Comment:
let's use 4.2 everywhere, I'm going to merge this as a bug fix

Reply:
Done